data/fedora-update.yaml

```diff
@@ -541,6 +547,11 @@ python-halite:
         "Halite is officially retired."
 python-HTMLgen:
     dead-upstream: true
+python-igor:
+    note: |
+        The package is functional under python3, but the executable output
+        is  slightly nicer under python2. So it is on py2 on purpose
```

Comment:
What does "nicer" mean? How is the output different on different Python versions?

Reply:
This was the reply of the maintainer of the package in the mailing list: https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/P3WPHS57OHH26XBLWIGFY4BJD35RU2VO/